lib/Server.js

```diff
@@ -610,7 +623,7 @@ class Server {
           const pems = createCertificate(attrs);
 
           fs.writeFileSync(certPath, pems.private + pems.cert, {
-            encoding: 'utf-8',
+            encoding: 'utf8',
```

Comment:
`utf8` is right value https://nodejs.org/api/fs.html#fs_fs_writefilesync_file_data_options